CONTRIBUTING.md

```diff
@@ -119,6 +119,37 @@ If you want to tweak texts, the name or the intro of a checker, you need to add
 - Create a new `Topic` and fill in the attributes accordingly
 - Duplicate an existing Intro component file in `packages/client/src/intros`, change the content to your wishes and name it **exactly** as you named it in the previous step `intro`
 
+## PreQuestions
+
+PreQuestions are used to render custom questions that are outside of the `imtr-client` and the `checker` class. For example,`PreQuestionMultipleCheckers` is used to inform the user and make the text in OutcomeSection dynamic.
+
+### Adding a new custom PreQuestion
+
+- Let's start by making sure the answer your new PreQuestion can be saved to the Session Storage. Add a new key to `const defaultTopicSession` in [SessionContext.tsx](packages/client/src/SessionContext.tsx) (eg: `questionNew: undefined,`)
+- ESLint should now complain that "`questionNew` does not exist in type `TopicData`". (If not, please make sure you run this project correctly on localhost.) Add `questionNew?: AnswerValue;` to `TopicData` in [types.ts](packages/client/src/types.ts) to type support `questionNew` as `undefined` or `AnswerValue`. ESLinst stops complaining now.
+- Let's continue by duplicating [PreQuestionMultipleCheckers.tsx](packages/client/src/components/Question/PreQuestionMultipleCheckers.tsx) and naming it `PreQuestionNew.tsx`
+- To make sure the `answer` value is loaded from and stored in the Session Storage, change the `topicDataKey` const (in the top of the file) to `questionNew` (Please make sure this is the same string as you entered in `defaultTopicSession` in a previous step)
+- Edit `questionId`, `heading` and optionally `description` and `questionAlert` to you own wishes (in your new `PreQuestionNew.tsx` file) (`questionId` is also the key in the translation file [i18n/nl.ts](packages/client/src/i18n/nl.ts))
+- Now, add an entry to the enum `PreQuestionComponent` in [types.ts](packages/client/src/types.ts) (eg: `NEW_PREQUESTION, // Corresponds to PreQuestionNew.tsx`)
```

Comment:
Careful, this step should work once #857 is merged. In this branch `PreQuestionComponent` is still located in `config/index.ts`

Reply:
`develop` wasn't merged with this branch yet. Now it is